lib/node_modules/@stdlib/_tools/scripts/print_npm_install_deps

```diff
@@ -1,101 +0,0 @@
-#!/usr/bin/env node
```

Comment:
Removing this file is not desired. Even with the proposed changes in this PR, we may still have other use cases for this script.